pyproject.toml

```diff
@@ -19,6 +19,10 @@ dynamic = ["version", "readme"]
 all = [
     "sentence-transformers",
     "transformers",
+    "nltk",
+    "rouge_score",
+    "fuzzywuzzy",
+    "rapidfuzz",
```

Comment:
we need to document this- that you can install with `pip install ragas[all]` for these metrics